src/ale_interface.cpp

```diff
@@ -90,14 +90,12 @@ void ALEInterface::loadSettings(const string& romfile,
   // Seed random number generator
   if (theOSystem->settings().getString("random_seed") == "time") {
     cerr << "Random Seed: Time" << endl;
-    srand((unsigned)time(0));
-    //srand48((unsigned)time(0));
+    Random::seed((unsigned)time(0));
```

Comment:
time() takes in a pointer as argument. Please change to time(NULL). Also, can you cast to the correct unsigned type (uInt32)? The latter also applies elsewhere in this code.